cluster/pulumi/infra/package.json

```diff
@@ -5,7 +5,8 @@
     "@pulumi/auth0": "3.21.0",
     "@pulumi/kubernetes-cert-manager": "0.2.0",
     "@pulumiverse/grafana": "0.16.3",
-    "splice-pulumi-common": "1.0.0"
+    "splice-pulumi-common": "1.0.0",
+    "splice-pulumi-common-sv": "1.0.0"
```

Comment:
snuck in the change from https://github.com/hyperledger-labs/splice/pull/1196/commits